tests/test_fernet.py

```diff
@@ -122,6 +123,51 @@ def test_bad_key(self, backend):
         with pytest.raises(ValueError):
             Fernet(base64.urlsafe_b64encode(b"abc"), backend=backend)
 
+    def test_timestamp_ttl_overdue(self, monkeypatch, backend):
+        ts = "1985-10-26T01:20:01"
+        current_time = calendar.timegm(iso8601.parse_date(ts).utctimetuple())
+        monkeypatch.setattr(time, "time", lambda: current_time)
+
+        issued_ts = struct.pack('>Q', calendar.timegm(
+            datetime.datetime(1985, 10, 26, 1, 19, 1).utctimetuple()))
+
+        f = Fernet(base64.urlsafe_b64encode(b"\x00" * 32), backend=backend)
+        with pytest.raises(InvalidToken):
+            token = base64.urlsafe_b64encode(b"\x80" + issued_ts)
```

Comment:
Instead of trying to construct the token manually please just call `encrypt` on Fernet to get a real token.

Reply:
Fixed.